src/components/Dropdown/Dropdown.tsx

```diff
@@ -86,7 +87,7 @@ const OptionsContainer = styled(Div)`
     position: absolute;
     top: 100%;
     left: 0px;
-    max-height: 10rem;
+    height: 10rem;
```

Comment:
This dramatically changes the look for Dropdowns with only 2/3 options - are we assuming that they can't change the height either?

Reply:
Hmm, didn't consider that. Specifying the height of the container [looks to be necessary](https://github.com/petyosi/react-virtuoso/issues/307#issuecomment-787873104) for the scoll/window calculations. I'll look into handling a smaller number of options that would not otherwise fill up the container.

Reply:
Short lists do look a bit odd - and I can definitely see use-cases where we want the size of the options container to expand with options instead of scroll (and not be virtualized at all)

![image](https://user-images.githubusercontent.com/7891559/129129709-215d9665-df09-4380-8795-23f2a17c8b24.png)